.circleci/config.yml

```diff
@@ -23,10 +23,11 @@ jobs:
             ./scripts/go-mod-download.sh
       - run: "! go fmt ./... 2>&1 | read"
       - run:
-          name: golint
+          name: lint
           command: |
-            go get golang.org/x/lint/golint
-            golint ./...
+            go get github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Hmm, shall we use a pinned version of the tool, to have a better reproducibility and avoid occasional breakage?